tests/test_transforms.py

```diff
@@ -45,3 +45,35 @@ def test_normal_scale_loc_transform_sink():
     )[0]
 
     assert isinstance(res.owner.op, type(at.random.normal))
+
+
+def test_invgamma_to_exp():
+
+    srng = at.random.RandomStream(0)
+    c_at = at.scalar()
+    X_rv = srng.invgamma(1.0, c_at)
+
+    fgraph = FunctionGraph(outputs=[X_rv], clone=False)
+    res = KanrenRelationSub(invgamma_exponential).transform(
+        fgraph, fgraph.outputs[0].owner
+    )[0]
+
+    assert isinstance(res.owner.op, type(at.true_div))
+    assert isinstance(res.owner.inputs[1].owner.op, type(at.random.exponential))
```

Comment:
I'm a little concerned about the "longevity" of this approach (i.e. the use of `type(...)`), because it's always possible that `at.true_div` and/or `at.random.exponential` could be changed from `Op` instances to helper functions at any point in the future.

Reply:
I agree that many of the tests here are lacking. Since we know the moment of the HalfCauchy, and forward sampling is fast, we can check that the mixture gets the first two moments right.

Reply:
Those checks are fine; it's the approach to performing them that I'm concerned about.

Reply:
The only other thing I can think of is implement the exponential model as well and check that the structures are similar. Is that what you had in mind?

Reply:
> The only other thing I can think of is implement the equivalent exponential model and check that the structures are similar. Is that what you had in mind? I modified the test in this direction.

That's generally better, but, to alleviate my concern, one only needs to import `ExponentialRV` and use that instead of `type(at.random.exponential)`.

Reply:
Yeah. I ended up implementing the equivalent exponential model and checking that the structures are similar. This is ready for review.